vendor/cel.dev/expr/regen_go_proto.sh

```
@@ -0,0 +1,9 @@
+#!/bin/sh
+bazel build //proto/cel/expr/conformance/...
+files=($(bazel aquery 'kind(proto, //proto/cel/expr/conformance/...)' | grep Outputs | grep "[.]pb[.]go" | sed 's/Outputs: \[//' | sed 's/\]//' | tr "," "\n"))
```

Shellcheck (0.10.0)

[warning] In POSIX sh, arrays are undefined.

(SC3030)